src/Sarif.Driver/Sdk/CommonOptionsBase.cs

```diff
@@ -64,8 +65,13 @@ public class CommonOptionsBase
             'v',
             "sarif-output-version",
             HelpText =
-            @"The SARIF version of the output log file. Valid values are OneZeroZero and Current",
+            "The SARIF version of the output log file. Valid values are OneZeroZero and Current",
             Default = SarifVersion.Current)]
         public SarifVersion SarifOutputVersion { get; set; }
+
+        public Formatting GetFormatting()
```

Comment:
>GetFormatting [](start = 26, length = 13)

This could be just a property
```
public Formatting Formatting => PrettyPrint ? Formatting.Indented : Formatting.None;
```